doc/cli/tcpdump.md

```diff
@@ -0,0 +1,90 @@
+## Network Traffic Inspection
+
+`tcpdump` is useful tool for analyzing and diagnosing network problems.
+This document presents the limited feature set that exposed is in the
+CLI.  Administrator level users with shell access can use the full
+feature set, and is not described here.
+
+The following section is useful for understanding how to use the tool.
+A section called [Examples](#examples) follows that, which may be what
+you want to scroll down to.
+
+
+### Hardware Overview
+
+Using `tcpdump` effectively requires an understanding of how the
+underlying hardware works.  For a standard PC, or common SBC, the
+network cards (NICs) are usually connected directly to the CPU.
+
+                          .---------.
+                          |         +-- eth0
+                          |   CPU   |
+                          |         +-- eth1
+                          '---------'
+
+In this setup it is evident that traffic coming in on either eth0 or
+eth1 reach the CPU, i.e., running `tcpdump eth0` captures all traffic.
+
+However, on other types of networking hardware, e.g., dedicated switch
+core setups depicted below, the flow of network traffic will likely
+*not* pass through the CPU.  This depends of course on how the switch is
+set up, for instance if routing between all ports is enabled, each flow
+will reach the CPU, but in a plain switching setup it will not.
+
+                          .---------.
+                          |         |
+                          |   CPU   |
+                          |         |
+                          '----+----'
+                               |
+                               |
+                    .----------+---------.
+                    |                    |
+                 ---+                    +---
+                X1  |         SC         |  X3
+                 ---+                    +---
+                X2  |                    |  X4
+                    '--+--+--+--+--+--+--'
+                       |  |  |  |  |  |
+                      X5 X6 X7 X8 X9 X10
+
+So, running `tcpdump x1` in a switching setup, inside the CPU, the only
```

Comment:
I think `s/X/e/g` would be more in line with the typical system.

Reply:
OK!